nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -513,6 +513,13 @@ class OVResultMetatype(OVOpMetatype):
                                   OVConvolutionBackpropDataMetatype,
                                   OVMatMulMetatype]
 
+METATYPES_WITH_WEIGHT_PORT_ID = GENERAL_WEIGHT_LAYER_METATYPES + [OVAddMetatype]
+
+# Contains the operation metatypes for which bias can be applied.
+OPERATIONS_WITH_BIAS_METATYPES = [OVConvolutionMetatype,
+                                  OVConvolutionBackpropDataMetatype,
+                                  OVMatMulMetatype]
+
```

Comment:
Why should we move this into another `common.py`? ONNX contains there variables and file in the same way.

Reply:
In TF and OV we have modules that store groups of metatypes:
TF: https://github.com/openvinotoolkit/nncf/blob/develop/nncf/tensorflow/graph/metatypes/common.py
OV: https://github.com/openvinotoolkit/nncf/blob/develop/nncf/experimental/openvino_native/graph/metatypes/common.py

But it seems that this behavior is not aligned across backends.

Reply:
So, we need to update ONNX as well, right?

Reply:
It would be great. My motivation is to store all variables that define a group of meta types in one place. Also, this behavior should be aligned across at least that backends, which were recently added (OV, ONNX). I hope you support this.